flake.nix

```diff
@@ -46,8 +46,8 @@
                 nixd
                 nixfmt-rfc-style
 
-                nodejs_20
-                nodejs_20.pkgs.pnpm
+                nodejs_24
+                nodejs_24.pkgs.pnpm
```

Comment:
this didn't change pnpm major version 🤔

Reply:
`pnpm --version` displays packageManager specified version is exist 
I don't know the background, but removed in https://github.com/kachick/wait-other-jobs/pull/1145